Modules/Core/QuadEdgeMesh/test/itkQuadEdgeMeshEulerOperatorDeleteCenterVertexTest.cxx

```diff
@@ -83,8 +83,8 @@ itkQuadEdgeMeshEulerOperatorDeleteCenterVertexTest(int argc, char * argv[])
   }
   std::cout << "OK" << std::endl;
 
-  ITK_TEST_EXPECT_EQUAL((const std::string) "QuadEdgeMeshEulerOperatorDeleteCenterVertexFunction",
-                        (const std::string)deleteCenterVertex->GetNameOfClass());
+  ITK_TEST_EXPECT_EQUAL(std::string_view("QuadEdgeMeshEulerOperatorDeleteCenterVertexFunction"),
+                        std::string_view(deleteCenterVertex->GetNameOfClass()));
```

Comment:
Thanks for merging, @hjmjohnson! For the record, none of the recent versions of this test (pull request #4576, #4583, #4587) properly handle the possibility of `deleteCenterVertex->GetNameOfClass()` accidentally returning a null. I was hoping that the `std::string_view(const char*)` constructor would support a null char pointer as argument, but unfortunately it does not. Neither does the originally used `std::string(const char*)` constructor. It's undefined behavior 🤷 

If you just want to ensure that this `GetNameOfClass()` override has code coverage, you might as well just checks that the returned pointer is not null. For example as follows:

    ITK_TEST_EXPECT_TRUE(deleteCenterVertex->GetNameOfClass());

Instead of doing the string comparison check.

Reply:
Is `GetNameOfClass()` ever expected to return a `NULL` pointer? So that problem might be moot.

Reply:
> Is `GetNameOfClass()` ever expected to return a `NULL` pointer?

No, but unit tests allow us to express what we expect, and what we do not expect 😸 

Obviously it's easy for anyone to write a malicious `GetNameOfClass()` override that _does_ return NULL. 😸 😸